datahub-web-react/src/app/lineage/LineageTreeNodeAndEdgeRenderer.tsx

```diff
@@ -4,7 +4,7 @@ import { curveBasis } from '@vx/curve';
 import { LinePath } from '@vx/shape';
 import { TransformMatrix } from '@vx/zoom/lib/types';
```

Comment:
Any component with an 'And' in the name makes me sad

Reply:
yes lol